### PR TITLE
refactor(test): refine function create_container

### PR DESCRIPTION
**Pull request by xujihui1985**
add CreateOptions as para to function create_container for create container with different options
this is a followup PR for https://github.com/youki-dev/youki/pull/2923/

### CODE REVIEW

**Comment by xujihui1985** [comment]
@YJDoc2 this PR is to follow up the previous PR https://github.com/youki-dev/youki/pull/2923, remove the redundant function and fix the test as well

**Comment by YJDoc2** [comment]
Hey @xujihui1985 thanks! I'll try to get to this in this week.

**Comment by YJDoc2** [comment]
Also, would you be willing to wait for few other PRs to be merged before this? There are several other test PRs which are in review, and if possible, I'd like to merge them before and ask you to rebase on main after that so there are no tests with older impl. If you're fine, I'll ping you once this is ready to rebase.

**Comment by xujihui1985** [comment]
> Also, would you be willing to wait for few other PRs to be merged before this? There are several other test PRs which are in review, and if possible, I'd like to merge them before and ask you to rebase on main after that so there are no tests with older impl. If you're fine, I'll ping you once this is ready to rebase.

sure, no rush. take your time

**Comment by YJDoc2** [comment]
@xujihui1985 , please go ahead and rebase this on main. Once this is updated, I'll merge :+1:

**Comment by xujihui1985** [comment]
> @xujihui1985 , please go ahead and rebase this on main. Once this is updated, I'll merge 👍

I have rebased from main branch and fix all the tests

File: tests/contest/contest/src/utils/test_utils.rs
Comment (YJDoc2) [inline tests/contest/contest/src/utils/test_utils.rs:203]: This diff is wild! :joy:

**Review by YJDoc2 — APPROVED** [review]
LGTM! Thanks for your efforts!

**Pull request merged**
